taxi/views.py

```diff
@@ -40,15 +42,60 @@
     queryset = Car.objects.all().select_related("manufacturer")
 
 
-class CarDetailView(LoginRequiredMixin, generic.DetailView):
+class CarDetailView(LoginRequiredMixin, DetailView):
+    model = Car
+    template_name = "taxi/car_detail.html"
+    context_object_name = "car"
```

Comment:
The same goes for template_name. The default template name is 'taxi/car_detail.html', which is the same as the one you've set. So you can remove this line as well.